index.bs

```diff
@@ -188,7 +316,37 @@ Every [=Document=] has an associated [=ordered set=] of <dfn>previously reported
     When asked to [=mark paint timing=] given a [=Document=] |document| as input, perform the following steps:
     1. If the [=document=]'s [=Document/browsing context=] is not [=paint-timing eligible=], return.
     1. Let |paintTimestamp| be the [=current high resolution time=].
-    1. Let |reportedPaints| be the [=previously reported paints=] associated with |document|.
+    1. Let |paintedImages| be a new [=ordered set=]
+    1. Let |paintedTextNodes| be a new [=ordered set=]
+    1. For each |record| in |doc|'s [=images pending rendering=] list:
+        1. If |record|'s [=pending image record/request=] is fully decoded, then run the following steps:
```

Comment:
Does decoded mean [=completely available=]?

Reply:
I would maybe say "[=available=]" (https://html.spec.whatwg.org/#img-available) and ready to be painted?

Reply:
"available" only means that the dimensions are known; we might not have any pixel content at all. Maybe, I'd ideally like to define a new state, for when enough data is available to display the initial frame of an animated image, or the complete image if not animated.

(I can submit a PR to HTML for that and use [=completely available=] here until that lands. We might still need a note that this doesn't include decode time.